tests/introspection/iOS/iOSApiProtocolTest.cs

```diff
@@ -93,6 +93,10 @@ protected override bool Skip (Type type)
 				// These were bound by mistake, and they're gone in XAMCORE_4_0.
 				return true;
 #endif
+            case "SNClassificationResult": // Class is not being created properly
+                return true;
+            case "SNClassifySoundRequest": // Class is not being created properly
+                return true;
```

Comment:
alignment is incorrect (spaces instead of tabs?)

Reply:
oops sorry, hopefully now it should be good